### PR TITLE
Remove unworking color modifiers

### PR DESCRIPTION
**Pull request by Vidmapps**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
             <div class="right flex-and-center">
-                <div class="right-item hover-color"><img src="img/025-mail.svg" class="-mail object-fit-contain-and-pointer"></div>
-                <div class="right-item hover-color"><img src="img/notif.svg" class="notif object-fit-contain-and-pointer"></div>
-                <div class="right-item hover-color"><img src="img/009-four.svg"class="-four object-fit-contain-and-pointer"></div>                                                        
+                <div class="right-item"><img src="img/025-mail.svg" class="-mail object-fit-contain-and-pointer"></div>
+                <div class="right-item"><img src="img/notif.svg" class="notif object-fit-contain-and-pointer"></div>
+                <div class="right-item"><img src="img/009-four.svg"class="-four object-fit-contain-and-pointer"></div>                                                        
                 <div class="profile flex-and-center">
-                    <div class="drop-down hover-color"><i class="fas fa-chevron-down"></i></div>
-                    <div class="name hover-color">Luke Borwn</div>
```
Comment (alynioke) [inline index.html]: why removed here? it's text so hover effect should work here

Comment (Vidmapps) [reply index.html]: True!

Comment (alynioke) [reply index.html]: if true, i suspect you'll add it back?

File: index.html [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
-                <div class="right-item hover-color"><img src="img/025-mail.svg" class="-mail object-fit-contain-and-pointer"></div>
-                <div class="right-item hover-color"><img src="img/notif.svg" class="notif object-fit-contain-and-pointer"></div>
-                <div class="right-item hover-color"><img src="img/009-four.svg"class="-four object-fit-contain-and-pointer"></div>                                                        
-                <div class="profile flex-and-center">
-                    <div class="drop-down hover-color"><i class="fas fa-chevron-down"></i></div>
-                    <div class="name hover-color">Luke Borwn</div>
+                <div class="right-item"><img src="img/025-mail.svg" class="-mail object-fit-contain-and-pointer"></div>
+                <div class="right-item"><img src="img/notif.svg" class="notif object-fit-contain-and-pointer"></div>
+                <div class="right-item"><img src="img/009-four.svg"class="-four object-fit-contain-and-pointer"></div>                                                        
+                <div class="profile flex-and-center hover-color">
```
Comment (alynioke) [inline index.html]: not clear why you added this class on this div, as it has no text inside of it.. so will be useless.

Comment (Vidmapps) [reply index.html]: Well it has text inside 
`<div class="profile flex-and-center hover-color">
                    <div class="drop-down"><i class="fas fa-chevron-down"></i></div>
                    <div class="name">**Luke Borwn**</div>
                    
                </div>`

I guess it is better to put that hover into child  `name` class ?

Comment (alynioke) [reply index.html]: i agree, yes :) will be clearer to what it applies.

**Comment by Vidmapps** [comment]
As far as I understand,  I can Merge PR?

**Comment by alynioke** [comment]
yes, you can merge! 
weird i see i reviewed but it doesn't show in PR.. 
image

**Pull request merged**